Fixed broken links (#2740)
SoftwareAG/c8y-docs: content/microservice-sdk/java-bundle/ip-tracker-microservice.md

In the Device Management application, navigate to **Devices** > **All devices** and locate your microservice tracker.
Under **Tracking** you will see a map with the tracked locations.
You can also develop your own web application and customize a "Map" widget.

> Refer to [Application library](/2024/web/libraries/#application-library).

Reviewer: ```suggestion
For details, refer to the [Web SDK](/2024/web/introduction) documentation.
```

> For details, refer to the [Web SDK](/2024/web/introduction) documentation.

Reviewer: ```suggestion
For details, refer to the [Web SDK](/web/introduction) documentation.
```

In the Device Management application, navigate to **Devices** > **All devices** and locate your microservice tracker.
Under **Tracking** you will see a map with the tracked locations.

> You can also develop your own web application and customize a map widget.

Reviewer: ```suggestion
You can also develop your own web application and customize a "Map" widget.
```

> Refer to [Application library](/2024/web/libraries/#application-library).

Reviewer: The link in develop should not point to the 2024 documentation. Seems that this section has been removed from develop. @janhommes Where should this point to now?

Reviewer 2: I guess this should go to the overall web SDK documentation: https://cumulocity.com/docs/web/introduction/
